Project.toml

```diff
@@ -2,7 +2,7 @@ name = "RoME"
 uuid = "91fb55c2-4c03-5a59-ba21-f4ea956187b8"
 keywords = ["SLAM", "state-estimation", "MM-iSAM", "MM-iSAMv2", "inference", "robotics"]
 desc = "Non-Gaussian simultaneous localization and mapping"
-version = "0.20.1"
+version = "0.20.2"
```

Comment:
```suggestion
version = "0.21.0"
```

Reply:
wait, this will be a minor since we lifting the Julia 1.8 minimum compat requirement.  my mistake